lib/common/client/errors.rb

```diff
@@ -20,6 +20,7 @@ def initialize(message = nil, status = nil, body = nil)
       class NotAuthenticated < ClientError; end
       class Serialization < ClientError; end
       class ParsingError < ClientError; end
+      class ValidationError < ClientError; end
```

Comment:
hmm, i'm not so sure this ought to go in `common/client/errors` because I really can't see this being something other clients would use. It's a little bit confusing because ValidationError would usually be something we would associate with validations failing returning a 422 response to the user. Could we keep this in FL namespace for now?